config/config_params.go

```diff
@@ -211,6 +211,8 @@ type Config struct {
 	IpInIpMtu        int    `config:"int;1440;non-zero"`
 	IpInIpTunnelAddr net.IP `config:"ipv4;"`
 
+	AWSSrcDstCheck   string `config:"oneof(donothing,enable,disable);donothing;non-zero"`
```

Comment:
Shouldn't these be `DoNothing`, `Enable` and `Disable`? Should match the libcalico-go API.

Reply:
Most of the current configuration uses small-caps. Per our conversation, will match it to `libcalico-go`.